File: 윤현철/JAVA/ClassEx210113/src/com/YHC/ClassEx210113/Student.java
package com.YHC.ClassEx210113;

public class Student {
	private String name;
	private int age;
	private char gender;
	private String address;
	
	public Student(String name, int age, char gender, String address) {
		super();
		this.name = name;
		this.age = age;
		this.gender = gender;
		this.address = address;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}

	public char getGender() {
		return gender;
	}

	public void setGender(char gender) {
		this.gender = gender;
	}

	public String getAddress() {
		return address;
	}

	public void setAddress(String address) {
		this.address = address;
	}
	public void showStudentInfo(){
		System.out.println(name);
		System.out.println(age);
		System.out.println(gender);
		System.out.println(address);
		System.out.println("-------------------------");
	}
}

File: 이재창/CShap/20210215/P1/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace P1
{
    class Student
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public char Gender { get; set; }
        public string Address { get; set; }

        public Student(string name, int age, char gender, string address)
        {
            Name = name;
            Age = age;
            Gender = gender;
            Address = address;
        }

        public void ShowStudentInfo()
        {
            Console.WriteLine($"이름: {Name}");
            Console.WriteLine($"나이: {Age}");
            Console.WriteLine($"성별: {Gender}");
            Console.WriteLine($"주소: {Address}");
            Console.WriteLine();
        }
    }
}
